uint32 uid
uint8 cnt
float32 dis
float32 aoa_angle_horizontal
float32 aoa_angle_vertical
